ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py

```diff
@@ -2334,3 +2334,48 @@ def test_jax_lax_real(
         test_values=True,
         x=x[0],
     )
+
+
+# squeeze
+@st.composite
+def _squeeze_helper(draw):
+    shape = draw(st.shared(helpers.get_shape(), key="value_shape"))
+    valid_axes = []
+    for index, axis in enumerate(shape):
+        if axis == 1:
+            valid_axes.append(index)
+    return valid_axes
+
+
+@handle_frontend_test(
+    fn_tree="jax.lax.squeeze",
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=st.shared(helpers.get_shape(
+                            allow_none=False,
```

Comment:
Hi! Just one small lint error `ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:2355:29: E126 continuation line over-indented for hanging indent`

Reply:
Hi Zoe, thanks for the suggestion! I've fixed it.